src/main/scala/com/wavesplatform/http/NodeApiRoute.scala

```diff
@@ -49,7 +49,7 @@ case class NodeApiRoute(settings: RestAPISettings, blockchainUpdater: Blockchain
   @ApiOperation(value = "Status", notes = "Get status of the running core", httpMethod = "GET")
   def status: Route = (get & path("status")) {
     val (bcHeight, bcTime) = lastHeight().map { case LastBlockInfo(_, h, _, _, t) => (h, t) }.getOrElse((0, 0L))
-    val (stHeight, stTime) = lastState().map { case StateWriter.Status(h, t) => (h, t) }.getOrElse((0, 0L))
+    val (stHeight, stTime) = (stateReader().height, stateWriter.lastUpdated)
```

Comment:
There's th `trait BlockchainDebugInfo`, please use it this stuff, BlockchainUpdater and StateWriter shouldn't be here at all